docs/content/docs/tutorials/sqs-example.md

```diff
@@ -0,0 +1,269 @@
+Setup](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-eks-setup.html).
+For automated cluster creation using `eksctl`, see [Getting started with Amazon EKS -
+`eksctl`](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) and create your cluster with
+Amazon EC2 Linux managed nodes.
+
+### Prerequisites
+
+This guide assumes that you have:
+
+- Created an EKS cluster with Kubernetes version 1.16 or higher.
+- AWS IAM permissions to create roles and attach policies to roles.
+- AWS IAM permissions to send messages to a queue.
+- Installed the following tools on the client machine used to access your Kubernetes cluster:
+  - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) - A command line tool for interacting
+    with AWS services.
+  - [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) - A command line tool for working
+    with Kubernetes clusters.
+  - [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) - A command line tool for working with EKS
+    clusters.
+  - [Helm 3.8+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
+
+### Install the ACK service controller for SQS
+
+Log into the Helm registry that stores the ACK charts:
+```bash
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+```
+
+Deploy the ACK service controller for Amazon SQS using the [sqs-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/sqs-chart). Resources should be created in the `us-east-1` region:
+
+```bash
+helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/sqs-chart --version=v0.0.3 --generate-name --set=aws.region=us-east-1
+```
+
+For a full list of available values to the Helm chart, please [review the values.yaml file](https://github.com/aws-controllers-k8s/sqs-controller/blob/main/helm/values.yaml).
+
+### Configure IAM permissions
+
+Once the service controller is deployed, you will need to [configure the IAM permissions][irsa-permissions] for the
+controller to query the SQS API. For full details, please review the AWS Controllers for Kubernetes documentation for
+[how to configure the IAM permissions][irsa-permissions]. If you follow the examples in the documentation, use the value
+of `sqs` for `SERVICE`.
+
+## Create a SQS Queue
+
+Execute the following command to create a manifest for a basic SQS queue, with an inline policy with `SendMessage`
+permissions for the account owner, and submit this manifest to EKS cluster using kubectl.
+
+{{% hint type="info" title="Make sure environment variables are set" %}}
+If you followed the steps in the IAM permissions section above, the required environment variables `${AWS_REGION}` and
+`${AWS_ACCOUNT_ID}` are already set. Otherwise please set these variables before executing the following steps. 
```

Comment:
You should call out here that the `helm install` command above configured the controller to run in `us-east-1`, so that should be the value for `AWS_REGION`

Reply:
Good catch!